manifest.json

```diff
@@ -3,8 +3,8 @@
 	"id": "spip",
 	"packaging_format": 1,
 	"description": {
-		"en": "SPIP - publishing system for the Internet",
-		"fr": "SPIP - Système de publication pour l’Internet"
+		"en": "A CMS with a focus on collaborative edition and multilingualism",
+		"fr": "Un CMS avec conçu pour l'édition collaborative et le multilinguisme"
```

Comment:
Thanks for this commit, Aleks, but the word "avec" seems to be useless...

Reply:
Indeed, fixed it, cheers ! :+1: